docker-compose.yml

```diff
@@ -14,15 +14,15 @@ services:
       - SPRING_DATASOURCE_URL=jdbc:postgresql://fees-database:5432/fees_register
       - SPRING_DATASOURCE_USERNAME=fees_register
       - SPRING_DATASOURCE_PASSWORD=fees_register
-      - AUTH_IDAM_CLIENT_BASEURL=https://idam-test.dev.ccidam.reform.hmcts.net
+      - auth.idam.client.baseUrl=http://idam-api:5000
     ports:
      - 8182:8080
     depends_on:
       - fees-database
 
   fees-database:
-    build:
-      context: docker/database
+#    build:
```

Comment:
If not required please remove the commented lines

Reply:
Have removed the commented lines